addons/mail/static/src/core/common/thread_model.js

```diff
@@ -303,7 +303,7 @@ export class Thread extends Record {
     }
 
     get hasAttachmentPanel() {
-        return this.model === "discuss.channel";
+        return this.channel;
```

Comment:
I thought I saw a PR moving this getter to discuss channel model that was approved.
Watch out for conflicts.

Reply:
Done!